header.php

```diff
@@ -1,3 +1,15 @@
+<?php
+/**
+ * Header file for the Twenty Twenty WordPress default theme.
+ *
+ * @link https://developer.wordpress.org/themes/basics/theme-functions/
```

Comment:
I think that we may want to use https://developer.wordpress.org/themes/basics/template-files/#template-partials here

Reply:
Yup good catch, will sort this in a moment.